plugins/chanbackup.c

```diff
@@ -760,6 +760,128 @@ static struct command_result *json_restorefrompeer(struct command *cmd,
 				     	    NULL);
 }
 
+/* Returns True if the backup data is correct and
+ * returns false with the reason in `fail` if it's incorrect.
+ * Instead of using `decrypt_scb()` we're using this because
+ * it shuts down the node in case of any discrepancy and that
+ * is not what we want to happen when someone runs `verifybackup`. */
+static bool verify_bkp(struct command *cmd,
+					   u8 *filedata,
+					   char **fail) {
+
+	crypto_secretstream_xchacha20poly1305_state crypto_state;
+	u64 version;
+	u32 timestamp;
+	struct scb_chan **scb;
+
+	if (tal_bytelen(filedata) < ABYTES +
+	    HEADER_LEN)
+	{
+		*fail = "data is corrupted!";
+		return false;
+	}
+
+	u8 *decrypt_scb = tal_arr(tmpctx, u8, tal_bytelen(filedata) -
+				  ABYTES -
+				  HEADER_LEN);
+
+	/* The header part */
+	if (crypto_secretstream_xchacha20poly1305_init_pull(&crypto_state,
+							    filedata,
+							    (&secret)->data) != 0)
+	{
```

Comment:
only the function has the starting block (`{`) in the next line, the if and other statement no

Reply:
Done